src/plugins/Makefile.am

```diff
@@ -192,6 +193,20 @@ libgs_plugin_flatpak_user_la_SOURCES =			\
 libgs_plugin_flatpak_user_la_LIBADD = $(GS_PLUGIN_LIBS) $(FLATPAK_LIBS)
 libgs_plugin_flatpak_user_la_LDFLAGS = -module -avoid-version
 libgs_plugin_flatpak_user_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
+
+libgs_plugin_external_apps_la_SOURCES =		\
```

Comment:
I wonder if the plugin shouldn't instead be called `eos-external` or something to that extent